plugin_test.go

```diff
@@ -37,6 +37,17 @@ func testPlugin(t *testing.T, pool *dockertest.Pool) {
 	f, err := os.Create(filepath.Join(t.TempDir(), "output.txt"))
 	assert.NoError(t, err)
 
+	defer func() {
+		err := os.RemoveAll(f.Name())
```

Comment:
This is not required since I was using `t.TempDir()` and that entire directory gets removed once the test finishes.